pkg/kopia/errors/errors.go

```diff
@@ -12,15 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package kopia
+package errors
```

Comment:
This package by itself could have been a separate PR to be merged independently. No changes required now 👍🏼